app/src/main/java/com/android/sample/mlUtils/ExerciseFeedBack.kt

```diff
@@ -159,22 +164,26 @@ class ExerciseFeedBack {
         onSuccess: () -> Unit = {},
         onFailure: () -> Unit = {}
     ): ExerciseCriterion {
+      val mult = 2f
```

Comment:
Good use of a constant, maybe we could name it in uppercase since it's the convention, but I don't think we did it for this project, so this is fine